src/adv.py

```diff
@@ -33,19 +36,105 @@
 room['narrow'].n_to = room['treasure']
 room['treasure'].s_to = room['narrow']
 
+# Item 
+maps = Item("map", "Old and faded")
+rag = Item("rag", "Dirty old")
+flashlight = Item("flashlight", "Cracked")
+compass = Item("compass", "Rusty")
+
+gold = Item("golden statue", "Beautiful, dazzling")
+
+# Appending items to the rooms
+room['outside'].items.append(maps)
+room['narrow'].items.append(rag)
+room['foyer'].items.append(flashlight)
+room['overlook'].items.append(compass)
+room['treasure'].items.append(gold)
+
+
 #
 # Main
 #
 
+def divider():
+    print(f"========================================\n")
+
+def adventrue_game():
+        print("Welcome to 'The Adventure Game'!")
+
+if __name__ == '__main__':
+    adventrue_game()
+
+
+def current_room():
+    print(text_wrapper.wrap(f"{player.current_room.description}"))
+    divider()
+
+def items():
+    print(f'Item: {player.current_room.items}')
+
+divider()
+
 # Make a new player object that is currently in the 'outside' room.
+name = input("Please enter your name:" )
```

Comment:
@jadeli1720 , I REALLY loved the greeting and the instructions you provided for your user! When I was playing your game, I tried to pick up an item and tried first by typing the item out, but it printed out a message that said I couldn't go that way. Perhaps you could continue improving the UX and provide specific instructions to the user that would help them overcome obstacles when they get stuck! Great work nonetheless! 👍 
![Screen Shot 2019-12-09 at 9 53 44 AM](https://user-images.githubusercontent.com/16110931/70445804-1fc3c280-1a6a-11ea-98de-97ff8815e2ce.png)